tests/test_matplotlib.py

```diff
@@ -1,10 +1,16 @@
 import unittest
 import os.path
 
+from distutils.version import StrictVersion
+
+import matplotlib
 import matplotlib.pyplot as plt
 import numpy as np
 
 class TestMatplotlib(unittest.TestCase):
+    def test_version(self):
+        self.assertLess(StrictVersion(matplotlib.__version__), StrictVersion("3.8.0"))
```

Comment:
Can we comment why we need this less than this version?

Reply:
d.